contracts/core/modules/IssuanceOrderModule.sol

```diff
@@ -513,13 +515,15 @@ contract IssuanceOrderModule is
     )
         private
     {
-        // Send left over maker token balance to taker
-        ITransferProxy(transferProxy).transfer(
-            _order.makerToken,
-            _requiredMakerTokenAmount.sub(_makerTokenUsed), // Required less used is amount sent to taker
-            _order.makerAddress,
-            msg.sender
-        );
+        // Send left over maker token balance to taker, if greater than 0
+        if (_requiredMakerTokenAmount.sub(_makerTokenUsed) > 0) {
```

Comment:
We could put `_requiredMakerTokenAmount.sub(_makerTokenUsed)` into a variable since its being used twice.

Reply:
+1